src/libraries/Common/tests/System/TimeProviderTests.cs

```diff
@@ -140,7 +140,7 @@ public void TestProviderTimer(TimeProvider provider, int MaxMilliseconds)
                             state,
                             TimeSpan.FromMilliseconds(state.Period), TimeSpan.FromMilliseconds(state.Period));
 
-            state.TokenSource.Token.WaitHandle.WaitOne(60000);
+            Assert.True(state.TokenSource.Token.WaitHandle.WaitOne(Timeout.InfiniteTimeSpan));
```

Comment:
The assert here is meaningless if an infinite timeout is used, as WaitOne only returns false if it times out.

Reply:
Ok, I removed the assert and kept the timeout to be infinite.